packages/terra-form-select/src/_Frame.jsx

```diff
@@ -726,7 +726,7 @@ class Frame extends React.Component {
           <span id={labelId}>{this.ariaLabel()}</span>
           <span id={descriptionId}>{this.renderDescriptionText()}</span>
         </div>
-        <div className={cx('display')} role={variant === Variants.DEFAULT ? 'textbox' : null}>
+        <div className={cx('display')} aria-label={this.props.intl.formatMessage({ id: 'Terra.form.select.display' })} role={variant === Variants.DEFAULT ? 'textbox' : null}>
```

Comment:
We should see if we can pull the ariaLabel prop, if it is undefined, default to the search string. I believe we have translations for the search string already.

Reply:
Fixed here https://github.com/cerner/terra-core/pull/2510/commits/b0d2c7642f0f8573835ac9235f89ca58b4001ffe.